src/PrestaShopBundle/Resources/views/Admin/layout.html.twig

```diff
@@ -59,7 +59,7 @@
         <div class="flash-message-list alert alert-danger">
             <ul>
             {% for flashMessage in app.session.flashbag.get('error') %}
-                <li>{{ flashMessage }}</li>
+                <li>{{ flashMessage|raw }}</li>
```

Comment:
for the record: allow HTML content in `flashBag` messages. As this is an internal feature, not usable from modules, it's not a securtity issue.

Reply:
Perhaps put that as a comment for future devs (and for @rGaillard :) ) ?